Unable to run flutter_embedding example

**Description**

<!-- A clear and concise description of what the problem is. -->

```
 flutter_embedding git:(main) jaspr serve
[CLI] Running jaspr in server rendering mode.
[BUILDER] [ERROR] Because jaspr_builder ^0.15.1 depends on jaspr ^0.15.1 and flutter_embedding_demo depends on jaspr ^0.16.0, jaspr_builder ^0.15.1 is forbidden.
[BUILDER] [ERROR] So, because flutter_embedding_demo depends on jaspr_builder ^0.15.1, version solving failed.
[BUILDER] [ERROR] You can try the following suggestion to make the pubspec resolve:
[BUILDER] [ERROR] * Try upgrading your constraint on jaspr_builder: dart pub add dev:jaspr_builder:^0.16.1
Bad state: Unable to start build daemon.
```

**Additional Context**

<!-- Add any other context about the problem here. -->

--- a/examples/flutter_embedding/pubspec.yaml
+++ b/examples/flutter_embedding/pubspec.yaml
@@ -21,7 +21,7 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   flutter_lints: ^4.0.0
-  jaspr_builder: ^0.15.1
+  jaspr_builder: ^0.16.1
   jaspr_web_compilers: ^4.0.10
 
 flutter:

